container.go

```diff
@@ -20,6 +21,8 @@ const (
 )
 
 type container struct {
+	sync.RWMutex
```

Comment:
Yeah, they are only protecting the handle, so it makes sense to name it as such. Uploaded fix.